src/django-app/django-project/shops/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+    
+from django.utils.translation import gettext as _
+
+class Shop(models.Model):
+    shop_owner = models.ForeignKey("accounts.ShopOwner", on_delete=models.CASCADE)
+    address = models.OneToOneField("Address", on_delete=models.PROTECT)
+    name = models.CharField(_("shop name"), max_length=255, unique=True)
+    num_employees = models.IntegerField(_("number of employees"), default=0, null=True, blank=True)
```

Comment:
Do we really need `num_employees` and `num_bays`? I think these should be calculated when needed by using a query like: `EmployeeData.objects.filter(shop=ShopInstance).count()`

Reply:
I think eventually, yes, this is a good idea. I was also thinking about doing something like that. For the purposes of the POC though, I kept them explicit so we didn't have to do any sort of calculations for it.

Reply:
```python
    @property
    def num_employees(self):
        return EmployeeData.objects.filter(shop=self.model).count()
```